src/serac/physics/utilities/functional/tests/functional_comparison_L2.cpp

```diff
@@ -0,0 +1,155 @@
+#include <fstream>
+#include <iostream>
+
+#include "mfem.hpp"
+
+#include "axom/slic/core/SimpleLogger.hpp"
+
+#include "serac/serac_config.hpp"
+#include "serac/numerics/mesh_utils.hpp"
+#include "serac/numerics/expr_template_ops.hpp"
+#include "serac/physics/operators/stdfunction_operator.hpp"
+#include "serac/physics/utilities/functional/functional.hpp"
+#include "serac/physics/utilities/functional/tensor.hpp"
+
+#include <gtest/gtest.h>
+
+using namespace std;
```

Comment:
I understand this is a test.  But having this many using's makes it very hard to read and understand where everything is coming from.  Can you remove the std and mfem one at least?